packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -176,6 +179,7 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
   handleDialogCancel = (e: Event | React.SyntheticEvent) => {
     _.invoke(this.props, 'onCancel', e, { ...this.props, open: false });
     this.setState({ open: false });
+    this.state.documentRef.body.style.overflow = this.state.prevOverflow;
```

Comment:
As @layershifter pointed out, there is no need for this to be in the state. Plus the `documentRef` may be undefined, so add a check for it

Reply:
I put it on the state to avoid using `(documentRef.current as HTMLDocument)` all the time... But, I am reming it then

Reply:
Done